src/Client.php

```diff
@@ -49,27 +49,6 @@ public static function test(RequestConfigInterface $requestConfig): void
         });
     }
 
-    /**
```

Comment:
@cerenozkurt Kullanılmayan collection ve interfaceleri silmemişsin? Emin olamadığın için mi?